master/master.cfg

```diff
@@ -309,13 +309,12 @@ for name, worker_name, buildfactory, stability, tier in BUILDERS:
     if "Windows7" in name:
         continue
 
-    buildername = name + " " + "PR"
+    buildername = f"{name} PR"
 
-    if stability != STABLE:
-        unstable_pull_request_builders.append(buildername)
-    else:
+    if stability == STABLE:
         stable_pull_request_builders.append(buildername)
-
+    else:
+        unstable_pull_request_builders.append(buildername)
 
     if any(pattern in buildername for pattern in NO_NOTIFICATION):
         continue
```

Comment:
I think if you axe this check, we'd land about where we want to be.

Reply:
I tried swapping the order of this check with adding the builder name to the list - that passes the config check!

it would work to axe it as well, but that would change the behavior for builders that match the patterns in `NO_NOTIFICATION`, and I don't know if that's a desirable change (I'm assuming there's a reason for that).

Reply:
with either change (swap order or axe it), I think either approach would work - do you have a preference which one to take?

1. add "stability" to the builder name and use it in the regex; or
2. the current state of this PR - use an explicit set of "stable builders" when not doing comment-based triggering

Reply:
I'd say 2.

I think we can do away with `NO_NOTIFICATION`.  The Alpine and UBSan builders are unstable, so should be left out of label builds anyway, and we no longer have a Cygwin builder.  I could see value in being able to manually trigger the Alpine or UBSan builders via a comment.

Reply:
Ok, scratch that: we can't quite do away with `NO_NOTIFICATION`, but we can drop the check on it here.

Reply:
ok, I dropped the check, I think this should be good to go!